CHANGELOG.md

```diff
@@ -4,16 +4,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [v1.0.0] - Unreleased
```

Comment:
are we going to v1.0.0, or v0.3.0?

Reply:
Could go either way, but I'm breaking some backwards compatibility, plus the charts seem to be in reasonable shape(?) to say there a version 1.

Reply:
I would suggest v0.3.0 until we put it through the motions a bit more :)